### PR TITLE
Package osbx.1.2.3

### PR DESCRIPTION
**Pull request by darrenldl**
### `osbx.1.2.3`

Implementation of SeqBox in OCaml

A single file container/archive that can be reconstructed even after total loss of file system structures (from official SeqBox).

---
* Homepage: https://github.com/darrenldl/ocaml-SeqBox
* Source repo: https://github.com/darrenldl/ocaml-SeqBox.git
* Bug tracker: https://github.com/darrenldl/ocaml-SeqBox/issues

### CODE REVIEW

File: packages/osbx/osbx.1.2.3/opam [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+authors: "Darren Ldl <darrenldldev@gmail.com>"
+homepage: "https://github.com/darrenldl/ocaml-SeqBox"
+bug-reports: "https://github.com/darrenldl/ocaml-SeqBox/issues"
+license: "BSD-3-Clause"
+dev-repo: "https://github.com/darrenldl/ocaml-SeqBox.git"
+build: ["jbuilder" "build" "@install"]
+depends: [
+  "jbuilder" {build}
+  "stdint" {build}
+  "nocrypto" {build}
```
Comment (samoht) [inline packages/osbx/osbx.1.2.3/opam]: are you sure that all these dependencies are `build` only?

**Comment by darrenldl** [comment]
@samoht You are absolutely right, I was being silly when writing the OPAM file. Thanks for pointing that out.

It should be correct now.

**Comment by darrenldl** [comment]
Actually I need to rethink about it.

This is supposed to produce a standalone binary, and I recall the produced binary ran successfully on systems without OCaml installation.

Maybe hold off processing this PR for the time being.

**Comment by darrenldl** [comment]
I can't think of a better way to express that in opam file.

I'll just leave the fixed opam file as it is right now(with only jbuilder being build dependency).

**Pull request merged**